prod/westeurope/internal/api/functions_admin/function_app/terragrunt.hcl

```diff
@@ -114,6 +114,7 @@ inputs = {
     USER_DATA_BACKUP_CONTAINER_NAME = dependency.storage_container_user-data-backup.outputs.name
     USER_DATA_DELETE_DELAY_DAYS     = 7
 
+    SLOT_TASK_HUBNAME = "ProductionTaskHub"
```

Comment:
it's not clear to me if we're creating a new function app or using the existing one.
in case we're creating a new function app we should introduce a block like 

```
resources_prefix = {
    function_app     = "fn3"
    app_service_plan = "fn3"
    storage_account  = "fn3"
  }
```

(only in this case we can change the task hub name here)

Reply:
This is the existing one: indeed for version 3 I've created a new story: https://www.pivotaltracker.com/story/show/174208031 
I'll work on it today. 
Since we are going to create a new instance for v3 we might not apply this change and wait for the v3.